spot-client/src/common/app-state/spot-tv/selectors.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * A selector which returns the last known room lock set on the Spot-TV's MUC,
+ * which must be entered to join the MUC.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {string}
+ */
+export function getCurrentLock(state) {
```

Comment:
When I first time see "lock" Im thinking about this like about a boolean, whether the room is locked or not. Would it be better to use password ? The XMPP spec names locked rooms "password protected", so it feels closer to what it really is

Reply:
not sure if just "password" or "mucPassword"

Reply:
or roomPassword

Reply:
Okay, we'll say mucPassword. I think at some point it won't be in redux, once the backend stuff gets made, so specifying it's for a muc is good. Plus it differentiates between a conference room, meeting room, and muc room.

Reply:
I'm gonna do this in a follow up PR once this PR lands and is stable. The naming is actually from existing logic and the term "lock" is used in a lot of places.